vllm/model_executor/layers/linear.py

```diff
@@ -751,10 +751,6 @@ def __init__(self,
             self.register_parameter("bias", None)
 
     def weight_loader(self, param: Parameter, loaded_weight: torch.Tensor):
-        # Special case for Fp8 scales.
-        fp8_scales_shard_indexer = getattr(param, "fp8_scales_shard_indexer",
```

Comment:
Looks like you accidentally included parts of your other PR here

Reply:
shoot. Will undo